### ISSUE TITLE
Tutorial feedback - Using Dojo 1 Dijits

### ISSUE THREAD

**Issue by umaar** [dojo/dojo.io#294]
This feedback is for this tutorial: https://dojo.io/tutorials/1040_dojo1_dijits/

---

__Feedback 1__

Step 0: “Dojo 2 based form widgets and with Dojo 1 Dijits”

Should give a basic overview of what a Dojo 1 ‘dijit’ is.

---

__Feedback 2__

Step 1: “Install additional development dependency:”

The visual spacing between headers is squashed:

---

__Feedback 3__

Step 1: “Install additional development dependency:”
“Dependency” -> “dependencies”

---

__Feedback 4__

Step 1: “Install additional development dependency:”

There should be some explanation into what the user is installing exactly.

---

__Feedback 5__

Step 2: “Some of the Dojo 1 typings reference typings from this library.”

Rephrase this?

---

__Feedback 6__

Step 2: “Add the Dijit module typings to the include section of tsconfig.json:”

This feedback is not specific to this tutorial. However I would prefer the code blocks in tutorials highlighted the line(s) I should be paying attention to so I don’t have to keep going back and forth to the code to understand what’s already present.

---

__Feedback 7__

Step 2:

“If you are using your own custom Dijits, you should add your own TypeScript typings to the project. The dojo/typings can provide you examples ”

“The dojo/typings can” -> “The dojo/typings [project/repo/thing] can”

---

__Feedback 8__

Step 3: “You should see a rather long output indicating many dependencies have been output.”

Funny usage of the word ‘output’ twice. Maybe ‘indicating many dependencies have been created’ ?

---

__Feedback 9__

Step 4: “To integrate Dijits within into a Dojo 2 application”

Grammar issue here, “within into”.

---

__Feedback 10__

Step 4: “needs to create a DOM stub to ”

More explanation on what is meant by a DOM stub.

---

__Feedback 11__

Step 4: “Now that you have wrapped Dijits for use, we need to import them into their containing Dojo 2 or modules.”

There is a missing term: “into their containing Dojo 2 [missing term here?] or”

---

__Feedback 12__

Step 5:

“We will be using w() function”.

“using “ -> “using the”

---

__Feedback 13__

Step 5: “ they behave very similar to Dojo 2 widgets”

Would replace this with “ they behave similarly to Dojo 2 widgets”

---

__Feedback 14__

Step 5: “Coupled with the dojo-typings TypeScript typings”

A little funny to read, maybe rephrase?

---

__Feedback 15__

Step 5: “represent their DOM virtually”

This links to a coming soon page.

---

__Feedback 16__

Step 5: “When we configured the externals for our”

Needs clarification, the “externals” what?

---

__Feedback 17__

Step 5: “Add theme class name to src/index.html:”

Rephrase this one, bit confusing.

---

__Feedback 18__

Step 5: “You should now have an application that will provide you with a form that”

When following along with the tutorial up to this point, I get this error in the console panel:

![image](https://user-images.githubusercontent.com/381495/31832062-7a5a6456-b5bd-11e7-808d-815a5a12e68a.png)

This error is also present in the “completed” demo application. When you create a “new worker”, the new worker does not include the first/last name:

![image](https://user-images.githubusercontent.com/381495/31832066-836b8fb6-b5bd-11e7-88b0-ed9683d73bc2.png)

I was trying to explore why this might be happening, and noticed if I comment out the parent Fieldset (a dijit) which the first name + last name  fields are part of, then the error disappears. 

![image](https://user-images.githubusercontent.com/381495/31832080-8e293188-b5bd-11e7-9c4d-ee206f5fc1bd.png)

The email field does not have this problem.

---

__Feedback 19__

Step 6: “In this tutorial, we learned how to setup the build ...”

In this paragraph, there are 3 “we learned” style notes, I think it’d be much clearer as a bullet pointed list.

**Comment by matt-gadd** [comment]
@umaar i think **Feedback 18** was due to this regression in widget-core https://github.com/dojo/widget-core/pull/724.
